data/xapleaux/xapleaux ships.txt

```diff
@@ -0,0 +1,199 @@
+ship Ascomy
+	sprite ship/ascomy
+	thumbnail thumbnail/ascomy
+	uncapturable
+	attributes
+		category "Heavy Warship"
+		cost 42140000
+		mass 945
+		bunks 138
+		"cargo space" 104
+		drag 22.3
+		"engine capacity" 44
+		"fuel capacity" 600
+		"heat dissipation" 0.82
+		hull 93800
+		"outfit space" 982
+		ramscoop 3
+		"required crew" 42
+		shields 32800
+		"solar collection" 88.4
+		"weapon capacity" 385
+		weapon
```

Comment:
```suggestion
		"X Ring Cradle" 1
		weapon
```

Unless I'm missing something, this ship needs an X Ring cradle in order to actually install that "Type X Coleo Engine", right?

Reply:
somehow it just works without giving any warning and in the game it has it equipped, maybe this prevents installing another one or removing it from the ship

Reply:
I swear I added that attribute but maybe I did so in the wrong file.

This is part of why I needed to get it into a draft, so I could actually get a clean set down where more than my eyes would be tracking the obvious things.

You are correct, the ships need to have the attributes. And two of these ships should actually have the attributes to support 2 different engines so yeah, there'll be some flexibility. Conversion outfits may even exist eventually.